src/main/cpp/cmake/get_flatbuffers.cmake

```diff
@@ -24,6 +24,7 @@ function(find_and_configure_flatbuffers VERSION)
     GIT_SHALLOW TRUE
```

Comment:
Need a copyright update in this file.

Reply:
cc @YanxuanLiu @pxLi to cover cmake files in the license header PR check.

Reply:
How could I forget this....